### ISSUE TITLE
Helium not playing local videos.

### ISSUE THREAD

**Issue by aali83** [JadenGeller/Helium#129]
I have been trying to play MP4 videos that are stored on my hard drive. It won't play them at all. There is no message. I had a Macbook air before and it was working. I am currently using 10.11.1

**Comment by brandonmowat** [comment]
I'm having the same issue when I try opening my .m4v files. I'm also running 10.11.1. What OS version was your Air running?

**Comment by aali83** [comment]
So 133 is closed because it's a duplicate of 129 and vice versa. Please open one of the defects. Still cannot play any local videos on helium, which is a big bummer.

**Comment by Fofer** [comment]
Local video playback (MP4 and M4V) is working fine for me in Helium.  I just drag them onto Helium's window, and they play.  I am running OS X 10.11.2.

**Comment by tanpet** [comment]
Good day. I downloaded Helium today and cannot play locally stored files. I have tried both .mp4 and .flv. Running OS X 10.10.5. Thanks in advance for any suggestions.

**Comment by Daskarzine** [comment]
Sorry, was there an answer to this? I just downloaded this very excitedly, but it won't play any local m4v files? Thanks

**Comment by Fofer** [comment]
That's strange, local .MP4 and .M4V and .FLV files are all playing fine for me with Helium v1.9.  I just tested files in all three formats.  Running OS X 10.11.2.

**Comment by brandonmowat** [comment]
This issue hasn't been resolved. i'm still not able to.

**Comment by Daskarzine** [comment]
Aah - maybe that's the issue, I downloaded version 2, you're on 1.9?!


On 9 Jan 2016, at 21:54, Fofer <notifications@github.com> wrote:

That's strange, local .MP4 and .M4V and .FLV files are all playing fine for me with Helium v1.9. I just tested files in all three formats. Running OS X 10.11.2.

**Comment by Fofer** [comment]
Yes, it appears that's the issue.  I just downloaded Helium v2.0 from the Mac App Store (didn't know there was an updated version there) and can confirm that the same  local .MP4 and .M4V and .FLV files do NOT play.  So I trashed it and went back to v1.9.

**Comment by Daskarzine** [comment]
Anyone know where I can download 1.9 from?

Thanks!

On 10 Jan 2016, at 01:40, Fofer <notifications@github.com> wrote:

Yes, it appears that's the issue. I just downloaded Helium v2.0 from the Mac App Store (didn't know there was an updated version there) and can confirm that the same local .MP4 and .M4V and .FLV files do NOT play. So I trashed it and went back to v1.9.

**Comment by KyleFromOhio** [comment]
Helium 2.0. Cant open .MP4 files locally. Nothing happens. Never plays.

**Comment by nruck1** [comment]
Version 1.9 works for playing local videos. You can find that version pretty easily with a google search or on macupdate com

NOTE: Don't update it & also check your VIDEO FILE FORMAT. MKV doesn't work with it, think limited to Mp4 and AVI (only tried MKV & Mp4 so far). If it doesn't load change the format.

**Comment by JohnKacz** [comment]
For those looking the easiest way to get 1.9 is here https://github.com/JadenGeller/Helium/releases

**Comment by AjGabriel** [comment]
I am also having the same issue and I am using helium version 2.0
![screen shot 2018-01-25 at 10 49 32 am](https://user-images.githubusercontent.com/10936392/35367968-0c8c0010-01bc-11e8-8bd5-5c8722beec35.png)

**Comment by slashlos** [comment]
For those interested, see [Helium 3 issue #4](https://github.com/slashlos/Helium/issues/4) noting a pre app store binary [zip](https://github.com/slashlos/Helium/files/1639956/Helium.zip) file.
